src/NexusMods.App.UI/Pages/Settings/SettingsView.axaml.cs

```diff
@@ -12,25 +13,38 @@ public SettingsView()
 
         this.WhenActivated(d =>
             {
-                this.BindCommand(ViewModel,
-                        viewModel => viewModel.SaveCommand,
-                        view => view.SaveButton)
+                this.BindCommand(ViewModel, viewModel => viewModel.SaveCommand, view => view.SaveButton)
                     .DisposeWith(d);
 
-                this.BindCommand(ViewModel,
-                        viewModel => viewModel.CancelCommand,
-                        view => view.CancelButton)
+                this.BindCommand(ViewModel,viewModel => viewModel.CancelCommand,view => view.CancelButton)
                     .DisposeWith(d);
 
-                this.BindCommand(ViewModel,
-                        viewModel => viewModel.CloseCommand,
-                        view => view.CloseButton)
+                this.BindCommand(ViewModel,viewModel => viewModel.CloseCommand,view => view.CloseButton)
                     .DisposeWith(d);
 
-                this.OneWayBind(ViewModel,
-                        viewModel => viewModel.SettingEntries,
-                        view => view.SettingEntriesItemsControl.ItemsSource)
-                    .DisposeWith(d);
+                this.WhenAnyValue(
+                    view => view.ViewModel!.SettingEntries,
+                    view => view.ViewModel!.Sections
```

Comment:
These are not expected to change right? This should not get triggered on every setting value change for example, right?